Makefile

```diff
@@ -203,13 +205,14 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-# Development clusters require linux/amd64 OCI image
-# Set platform to linux/amd64 regardless of host platform.
 # If using podman machine, and host platform is not linux/amd64 run
 # - podman machine ssh sudo rpm-ostree install qemu-user-static && sudo systemctl reboot
 # from: https://github.com/containers/podman/issues/12144#issuecomment-955760527
 # related enhancements that may remove the need to manually install qemu-user-static https://bugzilla.redhat.com/show_bug.cgi?id=2061584
 DOCKER_BUILD_ARGS ?= --platform=linux/amd64
```

Comment:
we could pass `--platform=linux/amd64,linux/arm64` and have simplified logic, but builds take longer

so, I think is best to use the fastest solution for development environment

Reply:
+1